Субмодули, исправления ошибок, доработка пунктов навигации

n2ref opened the pull request.

n2ref committed:
субмодуль метрик

n2ref committed:
Merge branch 'master' of https://github.com/easterism/core2

n2ref committed:
Новая версия планировщика

n2ref committed:
Добавлен иснталлер

n2ref committed:
В планировщике добавлена возможность просмотра текущего состояния вывода по задаче

n2ref committed:
Добавлен крон новой версии

n2ref committed:
Изменена сортировка с списке задач по названию

n2ref committed:
Исправление контрола switch

n2ref committed:
В планировщике добавлена время работы задачи

Archin20 committed:
Добавлены новые поля на странице детали part 1.0.1

Archin20 committed:
Merge remote-tracking branch 'origin/master'

n2ref committed:
Исправлен метод очистки логов

Archin20 committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
Модуль Auto Субмодуоль Parts
правки по слайдеру

n2ref committed:
Исправлена ошибка возникающая при работе с классом имеющем в составе своего имени Controller

n2ref committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
Merge remote-tracking branch 'origin/master'

n2ref committed:
Субмодули

n2ref committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
Merge remote-tracking branch 'origin/master'

n2ref committed:
Улучшено описание метода навигации

n2ref committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
Merge branch 'master' of https://github.com/shabuninil/core2

Archin20 committed:
Корзина заказов (добавление в корзину)

n2ref committed:
Добавлена возможность отдавать любые данные

Archin20 committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	mod/webservice/v2.1.0

n2ref committed:
Добавлена возможность использовать файлы в навигационном меню

Archin20 committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
Корзина заказов (страница корзины/страница заказа) -- правки 2.0
Добавлена возможность добавить в корзину из списка запчастей.

n2ref committed:
Обновление версии стилей

Archin20 committed:
Merge remote-tracking branch 'origin/master'

n2ref committed:
Добавлено обновление с версии 1.4.0

Archin20 committed:
Изменение статуса заказа для админа

n2ref committed:
Изменения стилей

n2ref committed:
Исправлена кнопка отправки сообщений

n2ref committed:
Исправлена ошибка при отправке писем

n2ref committed:
Убрано подтверждение перед очисткой кэша

n2ref committed:
Убрано подтверждение перед очисткой кэша

n2ref committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	mod/admin/js/admin.index.js

Archin20 committed:
Конструктор правил для импорта

Archin20 committed:
Merge remote-tracking branch 'origin/master'

n2ref committed:
Добавлены проверки при отправке ошибок и проверка при сохранении буфера в базу

Archin20 committed:
Merge remote-tracking branch 'origin/master'

n2ref committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
Форма подключения новых контрагентов в систему

n2ref committed:
Рефакторинг

n2ref committed:
Рефакторинг

Archin20 committed:
Форма подключения новых контрагентов в систему

n2ref committed:
Merge branch 'master' of github.com:Archin20/core2

 Conflicts:
	inc/classes/Init.php

n2ref committed:
Рефакторинг

easterism reviewed (commented):
Как сюда успела проникнуть регистрация??

Archin20 committed:
Исправление в восстановлении пароля

Archin20 committed:
УБрано ФИО из регистрации

n2ref committed:
Рефакторинг

n2ref committed:
Merge branch 'master' of github.com:Archin20/core2

 Conflicts:
	inc/classes/Init.php

n2ref committed:
Рефакторинг

Archin20 committed:
Merge pull request #1 from shabuninil/master

Рефакторинг

Archin20 committed:
добавлен прелодер

Archin20 committed:
добавлен прелодер

Archin20 committed:
- Создание пользователей в core_user после модерации
- Повторное отправление письма для продолжения регистрации (если не закончил регистрацию ранее)

Archin20 committed:
прелодер НА КНОПКАХ

n2ref committed:
Рефакторинг

Archin20 committed:
Исправлена Ошибка при открытии корзины

Archin20 committed:
Merge remote-tracking branch 'main/master'

# Conflicts:
#	html/material/login/index.html
#	html/material/login/indexMobile.html

Archin20 committed:
Изминение ссылок регистрации/восстановление пароля

n2ref committed:
Рефакторинг

n2ref committed:
Исправлена ошибка в регистрации

n2ref committed:
Исправлена ошибка в регистрации

n2ref committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	inc/classes/Init.php

Archin20 committed:
Merge remote-tracking branch 'main/master'

# Conflicts:
#	inc/classes/Init.php

Archin20 committed:
Merge remote-tracking branch 'main/master'

# Conflicts:
#	inc/classes/Init.php

Archin20 committed:
Поиск на главной для контрагента

Archin20 committed:
Перенесена настройка регистрационной формы и восстановления пароля в конфиг ядра

n2ref committed:
Изменена обработка уведомлений о заказах при импорте заказов

n2ref committed:
Рефакторинг

Archin20 committed:
Ссылки вида /restore/complete, /registration и прочие (исправление)

Archin20 committed:
Merge remote-tracking branch 'main/master'

# Conflicts:
#	html/material/login/index.html
#	html/material/login/indexMobile.html

Archin20 committed:
Добавлены проверки при регистрации восстановлении пароля
Сообщения с формы регистрации и восстановления приходят с бэка
В регистрации и восстановлении пароля убраны все алерты.

n2ref committed:
Рефакторинг

Archin20 committed:
Добавлены проверки при регистрации восстановлении пароля
Сообщения с формы регистрации и восстановления приходят с бэка
В регистрации и восстановлении пароля убраны все алерты.

Archin20 committed:
Merge remote-tracking branch 'main/master'

# Conflicts:
#	html/material/login/index.html
#	html/material/login/indexMobile.html

n2ref committed:
Merge branch 'master' of github.com:Archin20/core2

 Conflicts:
	html/material/login/index.html
	html/material/login/indexMobile.html

n2ref committed:
Рефакторинг

n2ref committed:
Рефакторинг

n2ref committed:
Добавлена версия ссылки ccs

Archin20 committed:
Merge remote-tracking branch 'main/master'

# Conflicts:
#	html/material/login/index.html
#	html/material/login/indexMobile.html

Archin20 committed:
Создание пользователя в момент ввода пароля

n2ref committed:
Перенос идентифкатора роли для регистрации в конфиг ядра

n2ref committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
В списке подтвержденных контрагентах нужно добавлен новый признак активности

n2ref committed:
Рефаткоринг

n2ref committed:
Улучшено формирование правил в ролях

n2ref committed:
Merge remote-tracking branch 'origin/master'

Archin20 committed:
Merge remote-tracking branch 'main/master'

Archin20 committed:
Модуль Admin
Доработка функционала автоматического проставления чекпоинтов по клику на тайтл модуля

n2ref committed:
Merge branch 'master' of github.com:Archin20/core2

Archin20 committed:
Вход контрагента в систему без модерации

Archin20 committed:
Merge remote-tracking branch 'main/master'

easterism commented on html/material/login/indexMobile.html:
```diff
@@ -19,10 +19,14 @@
+    <script src="core2/js/jquery/lib/jquery-1.12.4.min.js"></script>
+    <script src="core2/js/md5.js"></script>
+    <script src="core2/js/core-login.js"></script>
+    <script src="core2/html/material/js/sweetalert2.min.js?_=2"></script>
```
не должно подключаться, если не задана возможность регистрации

easterism commented on html/material/login/indexMobile.html:
```diff
@@ -19,10 +19,14 @@
+    <script src="core2/js/md5.js"></script>
+    <script src="core2/js/core-login.js"></script>
+    <script src="core2/html/material/js/sweetalert2.min.js?_=2"></script>
+    <script src="core2/js/jquery.maskedinput.min.js"></script>
```
не должно подключаться, если не задана возможность регистрации

n2ref replied:
уберем

n2ref replied:
убрано

n2ref committed:
Убран лишний sweetalert2 скрипт из логина

Archin20 committed:
Добавлена регистрация и восстановление пароля в остальные шкурки ядра

n2ref committed:
Регистрация и восстановление пароля перенесены во все шкурки

n2ref committed:
Описаны настройки

n2ref committed:
Изменены версии в статическом файле для скрипта login.css

n2ref committed:
В файлах, для мобильной версии, страницы Логин добавлено условие для добавления js скриптов

n2ref committed:
Исправлены нотисы

n2ref committed:
Добавлена поддержка старых версий приложений использующих webtoken

n2ref committed:
Добавлена связь контрагента по унп

n2ref committed:
Merge remote-tracking branch 'origin/master'

easterism commented on html/default/login/indexMobile.html:
```diff
@@ -4,11 +4,15 @@
 	<script type="text/javascript" src="core2/js/md5.js"></script>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 	<meta name="author" content="easter.by@gmail.com" />
 	<link rel="icon" type="image/png" href="[FAVICON_PNG]"/>
```
откуда опять [FAVICON_PNG]

n2ref replied:
убрано

easterism replied:
а в indexMobile.html

n2ref committed:
Исправление доступов в форме редактирования

n2ref committed:
В файлах мобильных версий исправлены фавиконки

n2ref committed:
В лист добавлена возможность получения sql условия используемого при поиске пользователем

n2ref committed:
Рефакторинг

n2ref committed:
Исправление ошибки

n2ref committed:
Добавлены файлы переводов на английский

n2ref committed:
Дополнен перевод на английский

easterism committed:
Update indexMobile.html
